Dockerfile

```diff
@@ -12,13 +12,15 @@ RUN yum -y install epel-release \
 ARG OMERO_VERSION=latest
```

Comment:
I'm assuming this has everyone's implicit :thumbsup: but now with 5.4 replacing 5.3